### ISSUE TITLE
Usage?

### ISSUE THREAD

**Issue by CanaryEarth** [Malvineous/ripper6#1]
Would it absolutely cause you so much pain that you instantly combust to include one teeny tiny usage example either here, or on the wiki, or anywhere else on the internet?

**Comment by Malvineous** [comment]
Not really, but nobody has ever asked and it works the same as every other command line program out there so I didn't think it was necessary: `./ripper6 file_to_search.xxx`
